### PR TITLE
Make graph more readable. Texts tweaked with shadows and bigger font …

### PR DESCRIPTION
**Pull request by fjcalzado**
…size and also increased repulsion force in particles

### CODE REVIEW

File: frontend/src/pages/search-page/components/graph/graph-view.business.ts [hunk 1]
```diff
@@ -91,38 +91,35 @@ const createNodes = (svg, graphDescriptor: GraphResponse, onGraphNodeDblClick: (
...
-const createNodeLabels = (svg, graphDescriptor: GraphResponse, onGraphNodeDblClick: (string) => void, theme: Theme) => {
+const createLabels = (svg, graphDescriptor: GraphResponse, onGraphNodeDblClick: (string) => void, theme: Theme) => {
   const ellipticalArc = `M${-5*nodeRadius},${0} A${5*nodeRadius},${2*nodeRadius} 0, 0,0 ${5*nodeRadius},${0}`;
   
-  const nodeLabelArcs = svg
+  const labelArcs = svg
     .append("g")
-      .attr("class", "nodelabelarcs")  
-    .selectAll(".nodelabelarc")
+      .attr("class", "labelarcs")
```
Comment (nasdan) [inline frontend/src/pages/search-page/components/graph/graph-view.business.ts]: style.labelarcs?

File: frontend/src/pages/search-page/components/graph/graph-view.handlers.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ const dragstarted = (simulation) => (d) => {
   if (!d3.event.active) simulation.alphaTarget(1).restart();
   d.fx = d.x;
   d.fy = d.y;
+  console.log(d);
```
Comment (nasdan) [inline frontend/src/pages/search-page/components/graph/graph-view.handlers.ts]: remove

File: frontend/src/pages/search-page/components/graph/graph-view.component.tsx [hunk 3]
```diff
@@ -10,9 +10,9 @@ import {
 } from "../../../../graph-api";
 import { cnc } from "../../../../util";
 
-
 const style = require("./graph-view.style.scss");
 
+
```
Comment (nasdan) [inline frontend/src/pages/search-page/components/graph/graph-view.component.tsx:15]: Remove extra space

File: frontend/src/pages/search-page/components/graph/graph-view.style.scss [hunk 4]
```diff
@@ -1,7 +1,48 @@
...
+    fill: $colorGrey;
+    stroke: darken($colorGrey, 50%);
+  }
+
+  .label {
+    cursor: pointer;
+    font-size: 1.15rem;
+    font-weight: 600;
+    text-shadow:
+      -1px -1px 2px #000,
```
Comment (nasdan) [inline frontend/src/pages/search-page/components/graph/graph-view.style.scss]: Why 8 values?

**Comment by nasdan** [comment]
Did you check on IE or Edge?

File: frontend/src/pages/search-page/components/graph/graph-view.handlers.ts (hunk 2 above)
Comment (fjcalzado) [reply frontend/src/pages/search-page/components/graph/graph-view.handlers.ts]: ups...

**Pull request merged**